### DISCUSSION TITLE
MessageBox.ShowError buttons

### DISCUSSION THREAD

**Discussion by RobertvanderHulst** [VsixCommunity/Community.VisualStudio.Toolkit#328] [Ideas]
The ShowError() message shows an error message with a [Ok] and [Cancel] button at this moment.
I think that in most cases an [Ok] button would be enough.
Do you all agree ?
Or should we add a ShowError() with a buttons argument with a default value of Ok+Cancel?

Robert

**Comment by StevenRasmussen** [comment]
Seems logical to me that an error message would only have an [Ok] button. My 2 cents.
